docs/azure/partials/common/_prerequisites.mdx

```diff
@@ -0,0 +1,18 @@
+### Azure Prerequisites
+
+For kubefirst to be able to provision your Azure cloud resources:
```

Comment:
```suggestion
For Kubefirst to be able to provision your Azure cloud resources:
```

Reply:
Whilst I agree that it should be a Kapital K, every other instance is lowerkase k. I've kept that for consistency, but I think we should perhaps update that outside of this PR.

Reply:
Will do as part of #773